sim/prng.ts

```diff
@@ -149,17 +246,6 @@ export class PRNG {
 		return out;
 	}
 
-	/**
-	 * The RNG is a Linear Congruential Generator (LCG) in the form: `x_{n + 1} = (a x_n + c) % m`
-	 *
-	 * Where: `x_0` is the seed, `x_n` is the random number after n iterations,
-	 *
-	 * ````
-	 * a = 0x5D588B656C078965
-	 * c = 0x00269EC3
-	 * m = 2^64
-	 * ````
-	 */
```

Comment:
Why is this comment being removed?

Reply:
Because it makes it really annoying to find anything in the file with 2x the large comments. I left the originals in on the main class. Perhaps that's worth reversing, but at the moment I had judged otherwise.